tools/SourceKit/tools/swift-lang/SourceKitdResponse.swift

```diff
@@ -17,12 +17,14 @@ import sourcekitd
 public class SourceKitdResponse: CustomStringConvertible {
 
   public struct Dictionary: CustomStringConvertible, CustomReflectable {
+    private let context: SourceKitdResponse
```

Comment:
Good catch! Could you add some comments why we need a strong reference to the response object?